### DISCUSSION TITLE
【日常】邦布修好了

### DISCUSSION THREAD

**Discussion by ykrsama** [share121/inter-knot#4773] [绝区零]
![122270386_p0](https://github.com/user-attachments/assets/4c4b03d2-9139-4422-8eb5-d6f4dd6154bc)

之前我的邦布充不进电，好几周没有开展绳匠业务了。拿到附近的邦布改装店去修，当天就修好了，开心<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by lllliufeng** [comment]
破事氵

**Comment by 206326402** [comment]
那个大叔太黑心，丁尼收费高，建议去远一点的外环修。。。

**Reply by Robin-txdy** [reply]
有推荐的店吗？我的邦布也坏了，正打算找店铺去修<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/569eb3011cb0253ca12055ed055ed0b4_2495493822724485444.png" alt="无奈呜呜" width="75" >
